database/packages.ts

```diff
@@ -1225,6 +1225,23 @@ export const laravelPackages: Package[] = [
         php_only: false,
         updated_at: '2023-04-04T14:09:48.903Z',
     },
+    {
+        name: 'Laravel Zero',
+        description: 'A micro-framework built on top of Laravel components for building command-line applications',
+        category: 'Code Architecture',
+        github: 'https://github.com/laravel-zero/laravel-zero',
+        author: 'Nuno Maduro and Owen Voke',
```

Comment:
Not sure what the author should be here, I just noticed they're github usernames (I think?), but this package has two authors mentioned in their readme

Reply:
The `author` should be the name of the composer package owner, So if it's `aimeos/aimeos-laravel` it becomes the first part before the slash: `aimeos`

Reply:
In some cases where there is no composer package, and it's a npm package (which does not indicate an owner from my understanding), you can use their github username in lowercase, or something they're famous for.
Like `https://github.com/xiCO2k/laravel-vue-i18n` which is a npm (`laravel-vue-i18n`) package the author becomes `xico2k`

Reply:
The idea is the `author` has to be unique so we can use it for filters and search and some other stuff